### DISCUSSION TITLE
no config.json file found

### DISCUSSION THREAD

**Discussion by BudsHub** [html2app/discussions#10] [General]
for the life of me I cannot see what Iam wrong, there is a fully configured json file in the archive.
I tried the example and that too has the same issue.

**Comment by yandeu** [comment]
It's my fault. The api keys have expired. I have to deploy the new keys first. Will do it tomorrow.

**Reply by yandeu** [reply]
I have made the update. It should work now.

**Reply by asaadbobby** [reply]
I am having this issue
